### PR TITLE
Updated HTML templates

### PR DESCRIPTION
**Pull request by jturnbull**
Added actual HTML to HTML templates.
Removed unnecessary spaceless and autoescape tags.

@incuna/backend please review

### CODE REVIEW

**Comment by jturnbull** [comment]
`{% spaceless %}` removes spaces between HTML tags. It doesn't seem a big deal for the HTML email templates (all HTML renders will do the same) and is meaningless in the context of the plain text email.

`{% autoescape off %}` essentially marks all the variables as safe, which doesn't appear to be a good idea.

File: user_management/api/templates/user_management/account_validation_email.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
-{% spaceless %}{% load i18n %}{% autoescape off %}{% blocktrans with name=site.name %}
-You are receiving this email because your email address has been used to
-register an account at {{ name }}.
+{% load i18n %}
+{% blocktrans with name=site.name %}
+<p>You are receiving this email because your email address has been used to
+register an account at {{ name }}.</p>
```
Comment (KevinEtienne) [inline user_management/api/templates/user_management/account_validation_email.html]: I'm wondering if we should split the template the other way around between the `p` tag and the blocktrans, eg:
```html
<p>
    {% blocktrans with name=site.name %}You are receiving this email...{% endblocktrans %}
</p>

<p>
    {% trans 'Please click the following link to complete your registration:' %}
</p>
```

Comment (jturnbull) [inline user_management/api/templates/user_management/account_validation_email.html]: Good idea - means the translation will be shared with the text file

**Comment by jturnbull** [comment]
@KevinEtienne updated

**Comment by jturnbull** [comment]
Thanks

**Pull request merged**